articles/openshift/tutorial-create-cluster.md

```diff
@@ -75,9 +75,9 @@ A Red Hat pull secret enables your cluster to access Red Hat container registrie
 
    You will need to log in to your Red Hat account or create a new Red Hat account with your business email and accept the terms and conditions.
 
-2. **Click Download pull secret.**
+2. Please go to the **[product page](https://developers.redhat.com/products/codeready-containers)** if it is the first time and after registration, it will take you to **[Red Hat OpenShift Cluster Manager page]** (https://cloud.redhat.com/openshift/) where you can Click **Download pull secret**.
```

Comment:
Go to the **[OpenShift product page](https://developers.redhat.com/products/codeready-containers)** if this is your first time creating a cluster. After registration, head to **[Red Hat OpenShift Cluster Manager page]** (https://cloud.redhat.com/openshift/) where you can click **Download pull secret** and download a pull secret to be used with your ARO cluster.